Add DXF extension

woutergd opened the pull request:
This PR adds the DXF output format extension to the WFS service. I already squashed the commits to a single commit. See also issue #531.

Known issues is that two tests are failing, but these were also failing for example on this PR: https://github.com/geoserver/geoserver-cloud/pull/541

groldan commented on src/apps/geoserver/wfs/pom.xml:
```diff
@@ -30,6 +30,11 @@
+    <dependency>
+      <groupId>org.geoserver.extension</groupId>
+      <artifactId>gs-dxf-core</artifactId>
+      <version>${gs.version}</version>
```
version is already set in the `src/pom.xml`, please remove it here

groldan commented on src/apps/geoserver/webui/pom.xml:
```diff
@@ -112,6 +112,11 @@
+    <dependency>
+      <groupId>org.geoserver.extension</groupId>
+      <artifactId>gs-dxf-core</artifactId>
+      <version>${gs.version}</version>
```
version is already set in the `src/pom.xml`, please remove it here

groldan reviewed (changes requested):
Hi @woutergd, thanks for the contribution. Looks good.
For the sake of completeness, would you mind adding both `gs-dxf-core` and `gs-dxf-wps` to the `wps-service`? looks to be equally straightforward, just need to add the dependencies and the corresponding locations to the `WpsApplicationConfiguration`'s `@ImportResource` annotation.

Also, have you, or can you, sign a GeoServer individual or corporate contribution license agreement?
Plase check the [CONTRIBUTING.md](https://github.com/geoserver/geoserver-cloud/blob/main/CONTRIBUTING.md) file for instructions, as we hope and invite you to continue contributing to the project :)

woutergd commented:
@groldan thanks for your feedback, I have updated the PR, and squashed already the changes. I haven't signed the contributor license yet, will make sure that that is submitted today.

woutergd commented:
@groldan Just to let you know, I just submitted a signed contributor license agreement.

groldan commented:
> @groldan Just to let you know, I just submitted a signed contributor license agreement.

excellent, thanks a lot.
